### DISCUSSION TITLE
Is AutoMapper compatible with .Net Framwork 4.8 ?

### DISCUSSION THREAD

**Discussion by hdahmani1** [LuckyPennySoftware/AutoMapper#4216] [NA]
Hi there, 
Can i use AutoMapper in .Net Framwork 4.8 environment ?

**Comment by jbogard** [comment]
Yes you can! All versions prior to 11.0 support .NET 4.8. Version 10.1.1 was the latest .NET 4.x-compatible release.

**Reply by hdahmani1** [reply]
Thank for yout replay. I tried 12.0.1 but did not work. I will try version 10.

**Reply by jbogard** [reply]
In the NuGet repository and website, you can examine the "Frameworks" section to see what the package targets.

**Reply by hdahmani1** [reply]
It works for version 10.1.1. Thank you.
